addons/analytic/tests/common.py

```diff
@@ -41,3 +42,23 @@ def setup_independent_company(cls, **kwargs):
     def get_default_groups(cls):
         groups = super().get_default_groups()
         return groups + cls.env.ref('analytic.group_analytic_accounting')
+
+    @classmethod
+    def setup_independent_user(cls):
+        default_groups = cls.env.ref('base.group_system') + cls.get_default_groups()
```

Comment:
Could you just put a comment explaining why we add the `group_system`? (company creation)